src/sealVerseData.ts

```diff
@@ -0,0 +1,10 @@
+import Option from 'components/Dropdown/Option'
+
+export default [
+  { label: 'SealCred', value: 'https://sealcred.xyz' },
+  {
+    label: 'SealCred Echo',
+    value: 'https://echo.sealcred.xyz',
+  },
+  { label: 'SealCaster', value: 'https://sealcaster.xyz' },
```

Comment:
This won't work on `localhost` isn't it? Should we use `window.location.origin`?